tests/test_save.py

```diff
@@ -263,8 +249,8 @@ def test_save_emmo_domain_ontology(
         "catalog-v001.xml",
     }
     assert set(
-        os.listdir(outputdir / "emmo.info" / "emmo" / "domain" / "chameo")
+        os.listdir(outputdir2 / "emmo.info" / "emmo" / "domain" / "chameo")
     ) == {"chameo.rdfxml", "catalog-v001.xml"}
-    assert set(
-        os.listdir(outputdir / "emmo.info" / "emmo" / "disciplines")
-    ) == {"isq.rdfxml", "catalog-v001.xml"}
+    assert not {"isq.rdfxml", "catalog-v001.xml"}.difference(
+        os.listdir(outputdir2 / "emmo.info" / "emmo" / "disciplines")
+    )
```

Comment:
Why this? It is more difficult to read.

Reply:
I agree that it is more difficult to read. But without this the test is failing since onto.save() creates other files in addition to isq.rdfxml and the catalogue file. 

We can either explicitly checking for all created files, but that will fail when new modules are added to EMMO disciplines.

Alternatively we could rewrite the test as

```python
created_files = set(os.listdir(outputdir2 / "emmo.info" / "emmo" / "disciplines"))
for fname in ("chameo.rdfxml", "catalog-v001.xml"):
    assert fname in created_files
```

Reply:
yes, I prefer this.

Reply:
done